mmv1/products/identityplatform/ProjectDefaultConfig.yaml

```diff
@@ -124,3 +124,136 @@ properties:
+ 
+  - !ruby/object:Api::Type::NestedObject
+    name: 'quota'
+    description: |
+      Configuration related to quotas.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'signUpQuotaConfig'
+        description: |
+          Quota for the Signup endpoint, if overwritten. Signup quota is measured in sign ups per project per hour per IP.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'quota'
+            description: |
+              Corresponds to the 'refill_token_count' field in QuotaServer config.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'startTime'
+            description: |
+              When this quota will take affect.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'seconds'
+              - !ruby/object:Api::Type::Integer
+                name: 'nanos'
+          - !ruby/object:Api::Type::NestedObject
+            name: 'quotaDuration'
+            description: |
+              How long this quota will be active for.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'seconds'
+              - !ruby/object:Api::Type::Integer
+                name: 'nanos'         
+           
+  - !ruby/object:Api::Type::NestedObject
+    name: 'recaptchaConfig'
+    description: |
+      The project-level reCAPTCHA config.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'emailPasswordEnforcementState'
+        description: |
+          The reCAPTCHA config for email/password provider, containing the enforcement status.
+          The email/password provider contains all related user flows protected by reCAPTCHA.
+        values:
+          - :RECAPTCHA_PROVIDER_ENFORCEMENT_STATE_UNSPECIFIED
+          - :OFF
+          - :AUDIT
+          - :ENFORCE
+      - !ruby/object:Api::Type::Array
+        name: 'managedRules'
+        description: |
+          The managed rules for authentication action based on reCAPTCHA scores.
+          The rules are shared across providers for a given tenant project.
+        item_type: Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Integer
+              name: 'endScore'
+              description: |
+                The end score (inclusive) of the score range for an action.
+            - !ruby/object:Api::Type::Enum
+              name: 'action'
+              description: |
+                The action taken if the reCAPTCHA score of a request is within the interval [start_score, end_score].
+              values:
+                - :RECAPTCHA_ACTION_UNSPECIFIED
+                - :BLOCK
+      - !ruby/object:Api::Type::Array
+        name: 'recaptchaKeys'
+        description: |
+          The reCAPTCHA keys.
+        item_type: Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'key'
```

Comment:
are these keys represented as a resource in another product?  They aren't ApiKeys I know, but if we can reference an object that is managed it will be less error prone.

Reply:
Ditto -- this field is removed from the PR for now.